src/foo_listenbrainz2.h

```diff
@@ -47,6 +47,15 @@ namespace lbz
 		return regex_search(mbid, rx);
 	}
 
+	static bool is_valid_token(const char* mbid)
+	{
+		if (mbid == nullptr) return false;
+		// 62 is the token length of Maloja. There's probably a better way to validate the token.
+		if (is_uuid(mbid) || (strlen(mbid) == 62 )) return true;
```

Comment:
It's been a while since I set up Maloja, but wasn't the token configured in a config file and could be set freely?

Reply:
Hmm. That could be. I configured it through on the web interface and it defaults to 62 chars. I haven't tested setting it in the config file.

Reply:
I also thought about adding a checkbox to enable and disable the custom API URL field in the settings, like the query field. That would also simplify the check for ListenBrainz vs. custom server, but complicate the preference pane logic a bit. But it would probably be a little more user friendly.

Reply:
Ah yes, there is a UI for this now. When I set up my instance the first time there was just that config file and I set any random string.

Agreed, let's do it like you say and just check for non-empty string in case it's not ListenBrainz

Reply:
I refactored the token and API URL check functions (see commit [5cc3739](https://github.com/phw/foo_listenbrainz2/pull/18/commits/5cc37395a2cfae0b1768117dce0ed9797aa37915)) and they seem to work correctly.